### DISCUSSION TITLE
Problem with time after converting into JSON

### DISCUSSION THREAD

**Discussion by Roky3029** [theOehrly/Fast-F1#506] [Q&A]
So, using `session = fastf1.get_session(2023, 4, "R")` and loading it, if I print it to the console, the total time it took the driver to complete the race is display. However, if I convert it to JSON it returns what I believe to be a unix timestamp, but after converting it to a human-readable format, it is converted into something totally different.
To give an example, if I fetch the 2023 Azerbaijan Grand Prix data, it shows that Perez completed the race in 1:32:42.436, but after converting it into json with `session.results.to_json()` it shows he completed the race in `5562436`, and if we convert that timestamp:

```py
>>> import datetime
>>> # Right below is the timestamp given by the API
>>> date = 5562436
>>> dt = datetime.datetime.fromtimestamp(date)
>>> print(dt)
1970-03-06 10:07:16
```

It may be sound like a stupid question, but I don't know if I'm doing the right thing to convert it into the same format shown before converting it to json

**Comment by Casper-Guo** [comment]
The 5562436 number is 1:32:42.436 in milliseconds, counting from the beginning of the race session.

`atetime.datetime.fromtimestamp` interprets it as seconds from the Unix epoch.

If you can clarify your intention with some desired input/output pairs, then we can suggest alternative implementations.

**Reply by Roky3029** [reply]
I'd just like to show the same thing that shows before converting the data to JSON.
So I'd be able to display on my web, for example:
Sergio Perez, 1st, 1:32:42.436.

**Reply by Casper-Guo** [reply]
```
from datetime import timedelta

delta = timedelta(milliseconds=5562436)
print(delta)
```

> 1:32:42.436000
